src/common/CMakeLists.txt

```diff
@@ -9,25 +9,28 @@
 ### Source files for common lib
 
 set(CLOUDSC_COMMON_SOURCES
-    module/parkind1.F90
     module/yoecldp.F90
     module/yomcst.F90
     module/yoethf.F90
     module/yoephli.F90
     module/yomphyder.F90
     module/routines.F90
-    module/abor1.F90
     module/timer_mod.F90
     module/mycpu.c
-    module/ec_pmon_mod.F90
-    module/oml_mod.F90
     module/file_io_mod.F90
     module/expand_mod.F90
     module/validate_mod.F90
     module/cloudsc_global_state_mod.F90
     module/cloudsc_mpi_mod.F90
 )
 
+set(CLOUDSC_FIAT_SOURCES
+    module/parkind1.F90
+    module/ec_pmon_mod.F90
+    module/oml_mod.F90
+    module/abor1.F90
+)
```

Comment:
My suggestion would be to roll back this change and restore these files in the list above.

```suggestion
```